configs/avo.json

```
@@ -1,6 +1,7 @@
 {
   "index_name": "avo",
-  "start_urls": ["https://www.avo.app"],
+  "start_urls": [],
+  "stop_urls": [],
   "sitemap_urls": [
     "https://www.avo.app/docs/sitemap.xml"
   ],
```

```suggestion
  "start_urls": ["https://avo.app"],
  "stop_urls": [],
  "sitemap_urls": ["https://avo.app/docs/sitemap.xml"],
  "force_sitemap_urls_crawling": true,
```

Hi,

I'm not sure why it doesn't work but you can force it and get `1896` hits this way.